plugins/module_utils/aci.py

```diff
@@ -1334,3 +1266,65 @@ def dump_json(self):
             if(output_path is not None):
                 with open(output_path, "a") as output_file:
                     json.dump([mo], output_file)
+
+    def call(self, method):
+        if method == 'GET':
+            call_path = self.path + self.filter_string
+            call_url = self.url + self.filter_string
+            data = None
+        if method == 'POST':
+            call_path = self.path
+            call_url = self.url
+            data = json.dumps(self.config)
+        if method == 'DELETE':
+            call_path = self.path
+            call_url = self.url
+            data = None
+        resp = None
+        if self.module._socket_path:
+            conn = Connection(self.module._socket_path)
+            info = conn.send_request(method, '/' + call_path, data)
```

Comment:
let's use format as we generally use so something like : '/{0}'.format(call_path)